tests/framework/extensions/workloads/template.go

```diff
@@ -87,3 +91,33 @@ func NewDaemonSetTemplate(daemonsetName string, namespace string, template corev
 		},
 	}
 }
+
+// NewJobTemplate is a constructor that creates a job template.
+func NewJobTemplate(jobName string, namespace string, template corev1.PodTemplateSpec) *batchv1.Job {
+	return &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: jobName,
+		},
+		Spec: batchv1.JobSpec{
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "",
```

Comment:
I've got a question for this empty string field; 

Could you verify on UI if it assigns this field an empty string while creating a workload/job?

I've got the same question for `ServiceAccountName` field.

Reply:
ServiceAccount is required as the user wont have permissions to run the commands otherwise.